tools/setup_network_scanner_config

```diff
@@ -69,25 +71,47 @@ fi
 # Enable the saned if "$SANED_CONFIG" is not the empty string otherwise disable the saned
 # and do the appropriate stuff regarding the xinetd:
 if [ -n "$SANED_CONFIG" ]
-then sed -i -e 's/^.*disable.*$/\tdisable     = no/' $XINETD_CONFIG_FILE || { echo "Failed to enable saned in $XINETD_CONFIG_FILE" 1>&2 ; exit 6 ; }
+then # First try the newest way (i.e. reload or start and enable SANED_SYSTEMD_UNIT)
+     # otherwise fall back to the older ways (i.e via xinetd):
+     if test -e $SANED_SYSTEMD_UNIT_FILE
+     then if systemctl --quiet is-active $SANED_SYSTEMD_UNIT && systemctl --quiet is-enabled $SANED_SYSTEMD_UNIT
+          then systemctl --quiet reload $SANED_SYSTEMD_UNIT || { echo "Failed to reload $SANED_SYSTEMD_UNIT" 1>&2 ; exit 7 ; }
+          else systemctl --quiet start $SANED_SYSTEMD_UNIT || { echo "Failed to start $SANED_SYSTEMD_UNIT" 1>&2 ; exit 8 ; }
+               systemctl --quiet enable $SANED_SYSTEMD_UNIT || { echo "Failed to enable $SANED_SYSTEMD_UNIT" 1>&2 ; exit 9 ; }
+          fi
+          exit 0
+     fi
```

Comment:
NP: Could it be separated to own function to make the bash hell a bit readable? This if ... then ... else is getting quite long and full of very similar code.

Reply:
Even the xinetd check is irrelevant for me once we dropped it. But as commented the script is directly the one from the attachment, so just helping @jsmeix for doing the Github PR.

From YaST we are just calling the script.

@jsmeix could you check @mchf comment and adapt it if needed?